.circleci/config.yml

```diff
@@ -581,6 +581,12 @@ workflows:
       - op-bindings-build:
           requires:
             - yarn-monorepo
+      - js-lint-test:
+          name: actor-tests-tests
+          package_name: actor-tests
+          dependencies: "(core-utils|sdk)"
```

Comment:
Should this also run on changes to `op-node`? Then we could see if there are performance changes. Perhaps even batcher/proposer

Reply:
The goal of this job is to test the testing framework rather than run the actor tests against the devnet. Right now it just runs the linter. We can add automated perf tests in the future, but that will require a different job and some updates to how actors work to collect additional metrics.